docs/pages/architecture/cost-optimizations.md

```diff
@@ -7,20 +7,21 @@ status: 'published'
 # Cost Optimizations
 
 Production feature engineering systems deal with lots of data at low latencies
-and hence can be very costly on cloud infrastructures. Fennel goes an extra
-mile in trying to keep cloud costs by squeezing as much out of hardware as 
-possible. This is accomplished by a long tail of cost optimizations -- many of
-which are too low ROI for each team to invest in individually but become feasible
-due to economies of scale enjoyed by Fennel.
+and hence can be very costly in terms of cloud infrastructure costs. Fennel 
+goes an extra mile in trying to keep cloud costs low by squeezing as much out 
+of cloud hardware as possible. This is accomplished by a long tail of cost 
+optimizations -- many of which are too low ROI for each team to invest in 
```

Comment:
too low "an" ROI or too low of an ROI. R is consonant but I believe "an" is the right article